hangman.py

```diff
@@ -0,0 +1,191 @@
+import os
+
+HANGMAN_ASCII_ART = ("""welcome to the game hangman /
+  _    _                                         
+ | |  | |                                        
+ | |__| | __ _ _ __   __ _ _ __ ___   __ _ _ __  
+ |  __  |/ _` | '_ \ / _` | '_ ` _ \ / _` | '_ \ 
+ | |  | | (_| | | | | (_| | | | | | | (_| | | | |
+ |_|  |_|\__,_|_| |_|\__, |_| |_| |_|\__,_|_| |_|
+                      __/ |                      
+                     |___/""")
+
+MAX_TRIES = (6)
+
+HANGMAN_PHOTOS = {"stage 0": "x-------x", "stage 1": """x-------x
+|
+|
+|
+|
+|""", "stage 2": """x-------x
+|       |
+|       0
+|
+|
+|""", "stage 3": """x-------x
+|       |
+|       0
+|       |
+|
+|
+""", "stage 4": """x-------x
+|       |
+|       0
+|      /|\\
+|
+|""", "stage 5": """x-------x
+|       |
+|       0
+|      /|\\
+|      /
+|""", "stage 6": """x-------x
+|       |
+|       0
+|      /|\\
+|      / \\
+|"""}
+
+def opening_screen():
+    """
+    the func prints the opening screen
+    :return void:
+    """
+    print(HANGMAN_ASCII_ART, "Your number of max tries is: {}".format(MAX_TRIES), sep='\n')
+
+def fix_path(file_path):
+    """
+    the funcs rempve
+    :param file_path:
+    :return:
+    """
+    if (file_path[0] == "\"" or file_path[0] == "\'") and (file_path[-1] == "\"" or file_path[-1] == "\'"):
```

Comment:
What does this do?
It seems like ur trying to strip quotes from the path, theres a better way to do it (strip())

Reply:
The strip() method in Python removes leading and trailing whitespace characters, which include spaces, tabs, and newline characters, from a string. It does not remove empty double quotes (""). If your string consists solely of empty double quotes, calling strip() will not remove them.

Reply:
nvm strip is good internet sucks